src/sentry/tasks/assemble.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import absolute_import, print_function
+
+import six
+import logging
+
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger(__name__)
+
+
+@instrumented_task(name='sentry.tasks.assemble.assemble_chunks', queue='assemble')
+def assemble_chunks(type, params, file_id, file_blob_ids, checksum, **kwargs):
+    '''This assembles multiple chunks into on File.
+    The type is a File.ChunkAssembleType
+    '''
+    if len(file_blob_ids) == 0:
+        logger.warning('sentry.tasks.assemble.assemble_chunks', extra={
```

Comment:
You can remove all of your log statements that are prepended with `sentry.tasks.assemble` since that will be in the logger name and is unnecessary.